desktop/cmp/form/FormField.js

```diff
@@ -47,8 +47,13 @@ export const [FormField, formField] = hoistCmp.withFactory({
     render({model, className, field, children, info, ...props}) {
 
         // Resolve FieldModel
-        const formContext = useContext(FormContext),
-            formModel = formContext.model;
+        const formContext = useContext(FormContext);
+        throwIf(
+            !formContext || (Object.entries(formContext).length === 0),
+            'Form field could not find valid FormContext. ' +
+            'Possibly caused by using an HTML form rather than a Hoist form.'
+        );
+        const formModel = formContext.model;
```

Comment:
+ . Doesn't seem like the Form or FormModel is necessary to Render (or should not be difficult to make it so). Think it a gentler development ex

+ Error message should be clearer, reference the actual package e.g.  "Make sure you are using a Hoist form ('@xh/hoist/cmp/form/Form')  and not an Html Form ('@xh/layout/Form`).

Reply:
I addressed those two things.

Reply:
Minor style point. would recommend using backticks here for the multi-line string with inner quotation marks.

Reply:
No need to go back to change this now, but the idea with the backticks is that you don't need to concat with `+` - you can include newlines in backtick quoted strings, so multi-line is cleaner.